packages/plugin-ext/src/plugin/plugin-context.ts

```diff
@@ -18,9 +18,8 @@ import { CommandRegistryImpl } from './command-registry';
 import { Emitter } from '@theia/core/lib/common/event';
 import { CancellationTokenSource } from '@theia/core/lib/common/cancellation';
 import { QuickOpenExtImpl } from './quick-open';
-import { MAIN_RPC_CONTEXT, Plugin } from '../api/plugin-api';
+import { MAIN_RPC_CONTEXT, Plugin as PPlugin, PluginManager } from '../api/plugin-api';
```

Comment:
could we use another name than PPlugin which might not be clear in the rest of the code